UnityProject/Assets/Scripts/Util/NetworkedLeanTween.cs

```diff
@@ -0,0 +1,182 @@
+﻿using UnityEngine;
+using Mirror;
+
+/// <summary>
+/// Handles syncing common variables used in animations.
+/// </summary>
+
+public class NetworkedLeanTween : NetworkBehaviour
+{
+	[SyncVar]
```

Comment:
SyncVars only work when the trigger is server, you have a couple calls using client side code.